src/hooks/useSuggestions.js

```diff
@@ -0,0 +1,95 @@
+import { useEffect, useMemo, useState } from 'react';
+import { collection, getDocs, query, orderBy, where } from 'firebase/firestore';
+
+import { SUGGESTIONS_STATUS } from 'constants/types';
+import { SUGGESTIONS } from 'firebase/collections';
+import firebaseData from 'firebase/firebase';
+import useAuth from 'hooks/useAuth';
+
+const { firebaseDb } = firebaseData;
+
+const useSuggestions = () => {
+  const DEFAULT_STATUS = useMemo(() => SUGGESTIONS_STATUS.find(({ isDefault }) => isDefault), []);
+  const [suggestions, setSuggestions] = useState([]);
+  const [filters, setFilters] = useState({ statusFilter: DEFAULT_STATUS });
+
+  const {
+    statusFilter: { value: statusFilterValue },
+  } = filters;
+
+  const {
+    user: { uid: myId },
+  } = useAuth();
+
+  useEffect(() => {
+    (async function () {
```

Comment:
i recommend you move this to a new method and then call it from here

Reply:
if im not wrong you already did it in a later PR

Reply:
yes I was going to say that. I did that in a later PR 😄